libs/wingsdk/src/shared-aws/util.ts

```diff
@@ -0,0 +1,23 @@
+import { Token } from "cdktf";
```

Comment:
> what would you recommend to do about the performance regressions (amazing that this was caught)?

Without doing any testing, my guess is that the issue is this. queue.ts imports this file, and having this at the top causes a `require("cdktf")` when you do `require("@winglang/sdk")`.

Try moving this inside isValidArn as a `require`